brands/amenity/cinema.json

```
@@ -295,8 +316,36 @@
       "name": "Haut et Court"
     }
   },
+
+  "amenity/cinema|Les Écrans de Paris": {
+    "locationSet": {
+      "include": ["fr"]
+    },
+    "tags": {
+      "amenity": "cinema",
+      "brand": "Les Écrans de Paris",
+      "brand:wikidata": "Q3236510",
+      "brand:wikipedia": "fr:Les Écrans de Paris",
+      "name": "Les Écrans de Paris"
+    }
+  },
+  "amenity/cinema|Groupe Majestic Cinemas": {
```

Can you change it to say "Majestic" here too? thanks!
```suggestion
  "amenity/cinema|Majestic": {
```